docs/core/core.adoc

```diff
@@ -30,14 +30,11 @@ Refer to the link:./getting-started[Getting Started] guide for more information
 
 == Usage
 
-The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK however:
+The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK:
 
 [source, javascript]
 ----
-import { ConfigurationHelper } from '@aerogear/core';
-import appConfiguration from '../mobile-services.json';
-
-const helper = new ConfigurationHelper(appConfiguration);
-const metricsConfig = helper.getConfigByType(MetricsService.TYPE);
+import { coreInstance } from '@aerogear/core';
```

Comment:
I think we should remove this part, it is not user docs, it is something for internal development.

Reply:
@josemigallas This will be done separately to get this out of the official docs: https://issues.jboss.org/browse/AEROGEAR-2692